### PR TITLE
Add "On" to some gossip methods in AI

### PR DESCRIPTION
**Pull request by illunix**
**Changes proposed:**
 
* Change from `GossipHello` to `OnGossipHello`, `GossipSelect` to `OnGossipSelect`, `GossipSelectCode` to `OnGossipSelectCode`, `QuestAccept` to `OnQuestAccept`, `QuestReward` to `OnQuestReward`,

**Target branch(es):**
 
* [x]  3.3.5
 
**Tests performed:**
 
Builds, tested ingame

Idk how to name prs and commits..

### CODE REVIEW

**Comment by kvipka** [comment]
why? just why?

**Comment by MaxtorCoder** [comment]
> 
> 
> why? just why?

Guess it's a preference type of thing.

File: src/server/scripts/World/go_scripts.cpp [hunk 1]
```diff
@@ -1545,4 +1545,4 @@ void AddSC_go_scripts()
     new go_darkmoon_faire_music();
     new go_pirate_day_music();
     new go_bells();
-}
+}
```
Comment (Keader) [inline src/server/scripts/World/go_scripts.cpp:1548]: missing end of file line

**Comment by Keader** [comment]
i realy deslike this, just cause conflicts with some custom code... for nothing

**Comment by illunix** [comment]
> i realy deslike this, just cause conflicts with some custom code... for nothing
What conflicts?

**Comment by MaxtorCoder** [comment]
> > i realy deslike this, just cause conflicts with some custom code... for nothing
> 
> What conflicts? @Keader 

Conflicts of custom code people have when they merge TC

**Comment by illunix** [comment]
Ah Yeah that make sense, so this PR will not be merged because of people custom scripts?

**Comment by MaxtorCoder** [comment]
> Ah Yeah that make sense, so this PR will not be merged because of people custom scripts?

And pretty much because it's more of a preference thing I guess

**Comment by illunix** [comment]
> i deslike dont mean that it should be closed.
> is just my point 👯

right

**Comment by jackpoz** [comment]
Thanks for the PR :)

**Comment by masterking32** [comment]
I always hate commits like this :D

**Comment by illunix** [comment]
Why?

**Comment by jackpoz** [comment]
@masterking32 we try to get things right the first time, sadly the initial name of these hooks was sGossopHello then changed to GossipHello (it should have been OnGossipHello from day one)

**Pull request merged**